external-task-worker/src/main/resources/application.yml

```diff
@@ -0,0 +1,7 @@
+micronaut:
+  application:
+    name: example-external-task-worker
+  server:
+    port: 8081
```

Comment:
The worker should not have a port. Is it possible to remove "netty" because we don't want an http server.